### ISSUE TITLE
The demo link in the documentation is broken

### ISSUE THREAD

**Issue by velum** [lukemadera/meteor-autoform-googleplace#27]
http://lukemadera-packages.meteor.com/af-googleplace-basic does not seem to exist anymore.

**Comment by shyaaam** [comment]
Yes, because meteor.com has pulled out their free meteor hosting platform so all meteor.com apps including this one will cease to work.
